src/pages/graphql/schema/product-recommendations/queries/recommendations.md

```diff
@@ -0,0 +1,240 @@
+                            "sku": "WP01",
+                            "rank": 5,
+                            "score": 41.731136,
+                            "queryType": "primary",
+                            "categories": [
+                                "women/bottoms-women",
+                                "women/bottoms-women/pants-women",
+                                "promotions/pants-all",
+                                "collections/performance-fabrics"
+                            ],
+                            "visibility": "Catalog, Search"
+                        }
+                    ]
+                }
+            ]
+        }
+    }
+}
+```
+
+## Input fields
+
+Field | Data type | Description
+--- | --- | ---
+`cartSKUs` | [String] | SKUs of the products in the cart.
+`category` | String | The category currently being viewed.
+`currentSKU` | String |  SKU of the product currently being viewed on the product detail page.
+`pageType` | PageType | Type of page on which recommendations are requested.
+`userPurchaseHistory` | [PurchaseHistory] | User purchase history with timestamp.
+`userViewHistory` | [ViewHistory] | User view history with timestamp.
+
+### PurchasedHistory
+
+Field | Data type | Description
+--- | --- | ---
+`date` | DateTime | Date of purchase
+`items` | [String]! | Items purchased
+
+### ViewHistory
+
+Field | Data type | Description
+--- | --- | ---
+`date` | DateTime | Date the items were viewed
+`sku` | [String]! | SKU of items viewed
+
+## Output fields
+
+The `Recommendations` object contains details about recommended products for a given SKU. It contains the following attributes.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`results` | [RecommendationUnit] | List of recommendation units for recommended products
+`totalResults` | Int | Total number of recommendation units for returned recommendation
+
+### RecommendationUnit attributes
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`displayOrder` | Int | Order in which recommendation units are displayed
+`pageType` | String | Page Type
+`productsView` | [ProductView] | List of product view
+`storefrontLabel` | String | Storefront label to be displayed on the storefront
+`totalProducts` | Int | Total number of products returned for a recommendation
+`typeId` | String | Type of recommendation
+`unitId` | String | Id of the preconfigured unit
+`unitName` | String | Name of the preconfigured unit
+
+### ProductView interface
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`categories` | [String] | List of categories to which the product belongs
+`rank` | Int | Rank given to a product
+`queryType` | String | Indicates if the product was retrieved from the primary or backup query
+`score` | Float | Score indicating relevance of the product to the recommendation type
+`sku` | String! | The product SKU
+`visibility` | String | Visibility setting of the product
+
+### SimpleProductView attributes
```

Comment:
SimpleProductView and ComplexProductView contain exactly the same fields as ProductView. I would say

The `ProductView` return object is an interface that can contain the following fields. It is implemented by the `SimpleProductView`  and `ComplexProductView` types. Both these types contain the same set of fields as `ProductView`.